docs/07-my-first-awesome.md

```diff
@@ -25,7 +25,7 @@ have not opened any programs. On the top right you see the time/date and a
 symbol showing the current layout. You can also click on the symbol to change
 the active layout.
 
-One of awesome's big advantages over other tiling window managers is its good
+One of awesome big advantages over other tiling window managers is its good
```

Comment:
You could write "One of the big advantages of Awesome" in case you want to get rid of the "'s".

btw: should be "Awesome" (capitalized) for consistency.